test/e2e/app-dir/router-stuck-dynamic-static-segment/next.config.js

```diff
@@ -2,6 +2,9 @@
  * @type {import('next').NextConfig}
  */
 const nextConfig = {
+  experimental: {
+    ppr: true,
+  },
```

Comment:
Is this intentional? I know our PPR testing uses an envvar to run tests without PPR with PPR and I presume that this is making this test fixture always run with PPR even in the non PPR test run